contracts/src/interfaces/Teleporter/IAvalancheICTTRouterFixedFees.sol

```diff
@@ -27,6 +36,26 @@ interface IAvalancheICTTRouterFixedFees is IAvalancheICTTRouter {
      */
     event UpdateRelayerFees(uint256 primaryRelayerFee, uint256 secondaryRelayerFee);
 
+    /**
+     * @notice Register a destination bridge for a token
+     * @param tokenAddress Address of the ERC20 token contract
+     * @param destinationChainID ID of the destination chain
+     * @param bridgeAddress Address of the destination bridge contract
+     * @param requiredGasLimit Gas limit requirement for sending to a token bridge
+     * @param isMultihop True if this bridge is a multihop one
+     * @param minPrimaryRelayerFee Minimal amount of tokens to pay as the optional Teleporter message fee
```

Comment:
Doesn't make sense to use "optional" anymore
```suggestion
     * @param minPrimaryRelayerFee Minimal amount of tokens to pay as the Teleporter message fee
```

Reply:
For sure.